iaac/terraform/aws-infra/rds/variables.tf

```diff
@@ -45,7 +45,7 @@ variable "db_allocated_storage" {
 variable "mysql_engine_version" {
   type        = string
   description = "The engine version of MySQL"
-  default = "8.0.30"
+  default = "8.0.31"
```

Comment:
Thanks for this PR! Can you use 8.0.32

Reply:
Yes, tested for that too. It's working fine

Reply:
I have updated the MySQL DB instance version for Cognito-RDS-S3 and RDS-S3 deployment.